client/js/views/GameView.vue

```diff
@@ -6,69 +6,132 @@
-                  key="opponent-hand-glasses"
-                  class="opponent-hand-wrapper transition-all"
-                  tag="div"
-                  name="slide-below"
-                >
-                  <card
-                    v-for="card in opponent.hand"
-                    :key="card.id"
-                    :suit="card.suit"
-                    :rank="card.rank"
-                    :data-opponent-hand-card="`${card.rank}-${card.suit}`"
-                    class="transition-all opponent-hand-card-revealed"
-                  />
-                </transition-group>
-                <transition-group
-                  v-else
-                  key="opponent-hand"
-                  tag="div"
-                  name="slide-below"
-                  class="opponent-hand-wrapper transition-all"
-                >
-                  <div
-                    v-for="(card, index) in opponent.hand"
-                    :key="index + 0"
-                    class="transition-all opponent-card-back-wrapper opponent-hand-card mx-2"
+        <template #prepend>
+          <v-list-item two-line>
+            <v-list-item-content>
+              <h3>History</h3>
+            </v-list-item-content>
+            <v-list-item-icon>
+              <v-icon color="neutral" large @click.stop="showHistoryDrawer = !showHistoryDrawer">
+                mdi-window-close
+              </v-icon>
+            </v-list-item-icon>
+          </v-list-item>
+        </template>
+
+        <v-divider />
+
+        <v-list dense>
+          <v-list-item v-for="(log, index) in logs" :key="index">
+            <p>
+              {{ log }}
+            </p>
+          </v-list-item>
+        </v-list>
+      </v-navigation-drawer>
+
+      <!-- Opponent Hand -->
+      <div class="opponent-hand-container">
+        <div
+          id="opponent-hand"
+          class="d-flex flex-column justify-start align-center px-2 pb-2 mx-auto"
+        >
+          <div class="user-cards-grid-container">
+            <username-tool-tip id="opponent-username-container" :username="opponentUsername" />
+            <div class="opponent-cards-container">
+              <div id="opponent-hand-cards" class="d-flex justify-center align-start">
+                <transition name="slide-below" mode="out-in">
+                  <transition-group
+                    v-if="hasGlassesEight"
+                    id="opponent-hand-glasses"
+                    key="opponent-hand-glasses"
+                    class="opponent-hand-wrapper transition-all"
+                    tag="div"
+                    name="slide-below"
                   >
-                    <v-card class="opponent-card-back" data-opponent-hand-card>
-                      <v-img :src="require('../img/logo_head.svg')" contain />
-                    </v-card>
-                  </div>
-                </transition-group>
-              </transition>
+                    <v-slide-group
```

Comment:
It seems a little weird to nest the `v-slide-group` inside the regular vue `<transition-group>` becuase I assume that vuetify injects a `<transition-group>` for its implementation of `<v-slide-group>`. Does this work okay?

Reply:
Oh interesting I didn't know that, It does seem to work without issue

Reply:
@itsalaidbacklife in trying move the `v-slide-group` out of the `transition-group`, the layout started acting a bit goofy actually. If you're cool with it, maybe we can revisit this since it's currently working as expected?

Reply:
Yeah that works for me